src/pages-and-resources/discussions/messages.js

```diff
@@ -70,6 +65,11 @@ const messages = defineMessages({
     defaultMessage: 'Partial support',
     description: 'A label indicating that an app only supports a subset of the possible features of a discussions app.',
   },
+  'appName.default': {
```

Comment:
Why this is required?

Reply:
Whoops - that shouldn't have made it into this PR.

Reply:
Fixed!

Reply:
Thank you 👍